### DISCUSSION TITLE
RefindPlus on Windows ARM64?

### DISCUSSION THREAD

**Discussion by JovanovicA** [dakanji/RefindPlus#187] [General]
### Suggested Feature

Is it possible that as win arm64 archicecture?, Thanks in advance.

### Background Information

Is it possible that as win arm64 archicecture?, Thanks in advance.

**Comment by dakanji** [comment]
Hello,

Converting to a discussion as there is just a question being asked. Please use the discussion feature when you just want to ask a question in future as suggested in the issue template.

**Comment by dakanji** [comment]
RefindPlus is currently x86 only and other architectures have not been looked at. 

Contributions to make it build and run on other architectures are welcome.
